.github/workflows/main.yml

```diff
@@ -0,0 +1,134 @@
+name: GH
+
+permissions:
+  contents: write
```

Comment:
This is for the release asset upload correct?  It could be scoped a little bit smaller to only that job

Reply:
a437409